docs/ipv4_forwarding.md

```diff
@@ -0,0 +1,32 @@
+## IPv4 Forwarding
+
+To enable PSO pods to communicate with each other across nodes in the cluster IPv4 forwarding must be enabled on all cluster nodes.
+
+If forwarding is not enabled PSO will fail to install and you will see a number of PSO related pods going into `CrashLoopBackoff` mode.
+
+### Check current IPv4 forwarding
+
+To check the current state:
+
+```bash
```

Comment:
is this command mistyped? should be checking it in /etc/sysctl.conf ?

Reply:
sysctl is the command - the second line is the output from the command

Reply:
Better to have a path before #, so it does not look like comments.

Reply:
I changed it to make it more obvious